src/vm/ast/parser/mod.rs

```diff
@@ -68,7 +68,7 @@ pub fn lex(input: &str) -> ParseResult<Vec<(LexItem, u32, u32)>> {
     let lex_matchers: &[LexMatcher] = &[
         LexMatcher::new(r##""(?P<value>((\\")|([[:print:]&&[^"\n\r\t]]))*)""##, TokenType::StringLiteral),
         LexMatcher::new(";;[[:print:]&&[^\n\r]]*", TokenType::Whitespace), // ;; comments.
```

Comment:
And this one?
`\n\r` to `\n`

Reply:
It shouldn't matter in either of these, though we should maybe clean up the regex (the `:print:` class is equivalent to `[ -~]`, which doesn't include any of `\n` `\r` or `\t`

Reply:
Addressed in: cab43a12a5a4395272d37c43d68610f47939aa5f

Reply:
Wait, I thought we wanted `\t` and `\n` as valid whitespace characters?  We at least want `\n`...

Reply:
that's a separate whitespace class

Reply:
(this one is for comments)